docs/sphinx/about/whats-new.txt

```diff
@@ -1,32 +1,32 @@
 Version history
 ===============
 
-5.3.0-m2 (2016 November 23)
----------------------------
+5.3.0-m3 (2016 December 8)
+--------------------------
 
-Updated build system:
+New features/API:
 
-* dropped embedded JARs and now use the Maven Ant Tasks plugin to unify the
-  dependencies using the pom.xml
-* improved Ant JAR and bundle target
-* dropped deprecated osgi targets, OME Tools bundle and ome-jxr component
-* removed PDF generation from the docs-sphinx target
-* added version number to Javadoc zip bundle name
-* migrated unit tests out of test-suite into formats-bsd
-* fixed test-suite targets, paths and symlink handling
-* fixed test-metadata and migrated it into test-suite
-* fixed mismatch between ``ND2HandlerTest`` package and location
-* cleaned up test-build to remove obsolete and decoupled components
-  and folders
-* simplified Travis build
-* POM repositories clean up to reduce complexity and use Maven Central as the
-  first location to look for dependencies
-* now storing all versions in the top-level pom.xml
-* updated build versioning from Maven by unified versioning strategy,
-  reviewing meta information stored in the manifests of each JAR and
-  introspecting this information in the ``FormatTools`` API to retrieve
-  version and revision numbers
-* updated developer documentation on updated build system
+* added support for JPEG-XR compressed CZI data (funded by a
+  `partnership between Glencoe Software and ZEISS <http://glencoesoftware.com/2016-08-30-glencoe-software-zeiss-partner-open-source-file-reader-whole-slide.html>`_), adding 'ome:jxrlib' as a new dependency
+  of Bio-Formats
+* tile-based image writing
+    - added new methods to the ``loci.formats.IFormatWriter`` interface
+      allowing to set and retrieve the tile along the X and Y dimensions
+    - added default implementations to the ``loci.formats.FormatWriter``
+      abstract class defaulting to the entire image width/height
+    - added functionality to ``loci.formats.TiffWriter`` allowing to write 
+      TIFF and derived formats like OME-TIFF to write tiled data
+* added ``loci.formats.DynamicMetadataOptions` a new implementation of the
+  ``loci.formats.MetadataOptions`` interface allowing to define options using
```

Comment:
Class and interface are both in `loci.formats.in` (for historical reasons I guess. They don't actually belong there because they also apply to writers).

However, I would not list the new class' name here, since we don't know if we're going to change it. How about "added a new MetadataOptions implementation that supports arbitrary key/value pairs"?